### DISCUSSION TITLE
Adding custom properties automatically (like trace_id and span_id)

### DISCUSSION THREAD

**Discussion by cchapman-kxs** [odygrd/quill#424] [Q&A] [answered]
Is there way to enrich logs automatically (without logging them explicity in each logging statement) for things like trace_id and span_id?  These are things that could be acquired at the call-site and would not be available in the logging thread.

**Comment by odygrd** [comment, marked as answer]
Currently the only way is to pass the trace_id and span_id for each log message for the reasons explained in https://github.com/odygrd/quill/discussions/426
